gobblin-modules/gobblin-orc/src/test/java/org/apache/gobblin/writer/GenericRecordToOrcValueWriterTest.java

```diff
@@ -27,6 +27,7 @@
 
 import org.apache.avro.Schema;
 import org.apache.avro.generic.GenericRecord;
+import org.apache.gobblin.util.orc.AvroOrcSchemaConverter;
```

Comment:
Do we need it? Same for the newly added line in `GobblinOrcWriter` and `GenericRecordToOrcValueWriter`

Reply:
yeah, basically I move class AvroOrcSchemaConverter from gobblin-orc module to gobblin-util module to make it's available for gobblin-hive-registration.